release_notes/ose_3_2_release_notes.adoc

```diff
@@ -619,6 +619,35 @@ behavior, run:
 $ oc rsh <name> -- /bin/bash
 ----
 
+[[browser-requirements]]
+== Browser Requirements
+
+ifdef::openshift-enterprise[]
+Review the https://access.redhat.com/articles/2176281[tested integrations] for
+{product-title}.
+endif::openshift-enterprise[]
```

Comment:
Well actually, you could drop the `ifdef` completely, since this is an enterprise-only topic overall.